platformio.ini

```diff
@@ -16,7 +16,7 @@ lib_deps =
   FastLED
 build_flags =
   -DJAZZLIGHTS_PLATFORMIO=1
-  -DWEARABLE=1
+  -DRENDERABLE=1
```

Comment:
So, I totally agree that wearable isn't relevant anymore. Renderable sounds funky though. Since you've looked through this, what are the cases where this macro is 0? Is it more a matter of old vs new?

Reply:
Great question, will check